wherehows-web/app/components/dataset-compliance.js

```diff
@@ -558,7 +565,7 @@ export default Component.extend({
    * Requires that the user confirm that any non-id fields are ok to be saved without a field format specified
    * @return {Boolean}
    */
-  confirmUnformattedFields() {
+  async confirmUnformattedFields() {
```

Comment:
The only thing is that this fn looks a little bit long, also it seems easy to extract pieces to make it more readable.